contents/blog/product-engineer-vs-product-manager.md

```diff
@@ -0,0 +1,75 @@
+tags: 
+    - Explainers
+---
+
+A product engineer is a combination of a software engineer and a product manager. We already compared [product engineers and software engineers](/blog/product-engineer-vs-software-engineer), and now is the time for product managers.
+
+Product engineers and product managers want to build a great product but differ in how they accomplish this. The product engineer ships code while the product manager researches, plans, and manages development. To understand both roles more, we:
+- compare how both work to build a product
+- explain how the rise of product engineers impacts product managers
+- explore how they can work together
+
+## How do product engineers and product managers build a product?
+
+Developing a product is a journey. The goal of the journey is to launch a successful product or feature that solves user problems. Both product engineers and managers share this goal. What is different is the path they take, which we walk through here.
+
+### Beginning the product development journey
+
+Both product engineers and managers are responsible for the start of the journey, figuring out what to work on. This means talking to users, analyzing usage data, understanding the competitive landscape, and using all this knowledge to prioritize work. Both translate user needs into product requirements.
+
+Product managers go into further depth here, often writing up reports and recommendations on their findings. This requires stronger analysis and research skills. Product engineers write issues and todos then move onto building as fast as possible.
+
+### Building the product
+
+Once product engineers and managers get a basic idea of what to work on, their paths begin to diverge. Product engineers dive quickly into building. They get their hands dirty writing code. They prototype and experiment. They ship previews, MVPs, and betas of features to get hands-on, direct feedback on the product.
+
+Product managers take more time before handing off work to engineers. They do further research into the business case for a product and understand where it aligns in the competitive landscape. They also spend time gaining depth into what the needs of users are and understanding their business and technology stack. 
+
+They use all this information to craft plans, requirements, and specs that solve the user needs they identified. They must translate this into something the whole team can understand. Product engineers face a similar task but have a smaller group to communicate with. Both roles require communication skills, but product managers place a higher emphasis on them.
+
+Once product managers completed their analysis and plans, they manage a team of "stakeholders" to get a product shipped. This can include design, marketing, ops, engineering, and more. They work with all of these teams to get the feature developed and launched. This ensures alignment across teams and lowers the risk of potential issues.
```

Comment:
```suggestion
Once product managers complete their analysis and plans, they manage a team of "stakeholders" to get a product shipped. This can include design, marketing, ops, engineering, and more. They work with all of these teams to get the feature developed and launched. This ensures alignment across teams and lowers the risk of potential issues.
```